src/diffusers/schedulers/scheduling_lms_discrete.py

```diff
@@ -202,22 +202,7 @@ def step(
 
         if isinstance(timestep, torch.Tensor):
             timestep = timestep.to(self.timesteps.device)
-        if (
-            isinstance(timestep, int)
-            or isinstance(timestep, torch.IntTensor)
-            or isinstance(timestep, torch.LongTensor)
-        ):
-            deprecate(
-                "timestep as an index",
```

Comment:
can we maybe leave this one until `0.8.0` - it hasn't been deprecated that long ago and is quite important IMO